library/src/main/java/com/google/maps/android/data/kml/KmlStyleParser.java

```diff
@@ -172,6 +174,10 @@ private static void setIconUrl(XmlPullParser parser, KmlStyle style)
     private static void setIconHotSpot(XmlPullParser parser, KmlStyle style) {
         float xValue, yValue;
         String xUnits, yUnits;
+        if (TextUtils.isEmpty(parser.getAttributeValue(null, "x")) ||
```

Comment:
It'd be better if an early return that checks if the method [XmlPullParser.isEmptyElementTag](https://developer.android.com/reference/org/xmlpull/v1/XmlPullParser.html#isEmptyElementTag()) is `true` so that the intent is clearer.

Reply:
Thanks, makes sense! I just made the change in https://github.com/googlemaps/android-maps-utils/pull/600/commits/edfcd62244466247968721878a51c30b67990b98.